libs/server/Resp/BasicCommands.cs

```diff
@@ -1463,13 +1463,13 @@ void ProcessHelloCommand(byte? respProtocolVersion, ReadOnlySpan<byte> username,
                 this.clientName = clientName;
             }
 
-            (string, string)[] helloResult =
+            (string, object)[] helloResult =
                 [
                     ("server", "redis"),
                     ("version", storeWrapper.redisProtocolVersion),
                     ("garnet_version", storeWrapper.version),
-                    ("proto", $"{this.respProtocolVersion}"),
-                    ("id", "63"),
+                    ("proto", this.respProtocolVersion),
+                    ("id", 63),
```

Comment:
Please fix `AsyncTest1` in `RespTests.cs` -- set `expectedNewlineCount = 30` since there are two fewer newlines in the output.

Reply:
Thanks! I'll try it!